24.md

```diff
@@ -17,7 +17,32 @@ These are extra fields not specified in NIP-01 that may be present in the string
   - `website`: a web URL related in any way to the event author.
   - `banner`: an URL to a wide (~1024x768) picture to be optionally displayed in the background of a profile screen.
   - `bot`: a boolean to clarify that the content is entirely or partially the result of automation, such as with chatbots or newsfeeds.
-  - `pronouns`: a string representing the preferred way to refer to this person.
+
+### User defined custom fields
+
+A `fields` attribute may also be added to profile metadata to represent user-defined fields. It is an array name-value tuples, eg:
+
+```json
+[["<name>", "<value>"], ...]
+```
+
+In the context of a full metadata object:
+
+```json
+{
+  "name": "Alex",
+  "picture": "https://...",
+  "fields": [
```

Comment:
maybe rename to `custom_fields` or `user_fields` instead of just `fields`, to make it immediately clear it's user defined?